srml/system/src/lib.rs

```diff
@@ -371,6 +376,17 @@ pub fn ensure_inherent<OuterOrigin, AccountId>(o: OuterOrigin) -> Result<(), &'s
 	}
 }
 
+/// An alias of ensure_inherent. Ensure that the origin `o` represents an unsigned extrinsic.
+///
+/// Returns `Ok` or an `Err` otherwise.
+/// This method is preferred in contexts where we have unsigned transactions being propagated
+/// in the network. The validity of the unsigned transactions should be ensured by the runtime.
```

Comment:
```suggestion
/// in the network. The validity of the unsigned transactions **MUST** be ensured by the runtime.  Otherwise, anyone could forge arbitrary unsigned transactions.
```